js/passwordManager/passwordManager.js

```diff
@@ -9,126 +9,122 @@ const Bitwarden = require('js/passwordManager/bitwarden.js')
+  return success
+}
 
-    return PasswordManagers.managers.find(mgr => mgr.name === managerSetting.name)
-  },
-  getConfiguredPasswordManager: async function () {
-    const manager = PasswordManagers.getActivePasswordManager()
+// Binds IPC events.
+function initialize () {
+  // Called when page preload script detects a form with username and password.
+  webviews.bindIPC('password-autofill', async (tab, args, frameId, frameURL) => {
+    const manager = await getConfiguredPasswordManager()
     if (!manager) {
-      return null
+      return
     }
 
-    const configured = await manager.checkIfConfigured()
-    if (!configured) {
-      return null
+    // it's important to use frameURL here and not the tab URL, because the domain of the
+    // requesting iframe may not match the domain of the top-level page
+    let hostname = new URL(frameURL).hostname
+
+    if (!manager.isUnlocked()) {
+      await unlock(manager)
     }
 
-    return manager
-  },
-  // Shows a prompt dialog for password store's master password.
-  promptForMasterPassword: async function (manager) {
-    return new Promise((resolve, reject) => {
-      const { password } = ipcRenderer.sendSync('prompt', {
-        text: l('passwordManagerUnlock').replace('%p', manager.name),
-        values: [{ placeholder: l('password'), id: 'password', type: 'password' }],
-        ok: l('dialogConfirmButton'),
-        cancel: l('dialogSkipButton'),
-        height: 175
-      })
-      if (password === null || password === '') {
-        reject(new Error('No password provided'))
-      } else {
-        resolve(password)
-      }
-    })
-  },
-  unlock: async function (manager) {
-    let success = false
-    while (!success) {
-      let password
-      try {
-        password = await PasswordManagers.promptForMasterPassword(manager)
-      } catch (e) {
-        // dialog was canceled
-        break
-      }
-      try {
-        success = await manager.unlockStore(password)
-      } catch (e) {
-        // incorrect password, prompt again
-      }
+    if (hostname.startsWith('www.')) {
+      hostname = hostname.slice(4)
     }
-    return success
-  },
-  // Binds IPC events.
-  initialize: function () {
-    // Called when page preload script detects a form with username and password.
-    webviews.bindIPC('password-autofill', function (tab, args, frameId, frameURL) {
-      // it's important to use frameURL here and not the tab URL, because the domain of the
-      // requesting iframe may not match the domain of the top-level page
-      const hostname = new URL(frameURL).hostname
-
-      PasswordManagers.getConfiguredPasswordManager().then(async (manager) => {
-        if (!manager) {
-          return
-        }
-
-        if (!manager.isUnlocked()) {
-          await PasswordManagers.unlock(manager)
-        }
-
-        var formattedHostname = hostname
-        if (formattedHostname.startsWith('www.')) {
-          formattedHostname = formattedHostname.slice(4)
```

Comment:
This breaks autofill on sites starting with `www.`, because we pass `hostname` in the autofill response, and it then makes this check fail:

https://github.com/minbrowser/min/blob/323e5dff0185be09a53dd558f891a5bdb373cfbe/js/preload/passwordFill.js#L273-L275

(We should keep the check, it's guarding against us accidentally sending IPC messages to the wrong tab)